### PR TITLE
Fix missing endpoint

### PR DESCRIPTION
**Pull request by PatStLouis**
Partially addresses #83.

Adds a try catch when fetching implementation endpoints.
Add an if condition if no endpoint is set when conducting a test (this is so the test suite doesn't mark it as passed when the call is rejected)

### CODE REVIEW

File: tests/TestEndpoints.js [hunk 1]
```diff
@@ -16,12 +16,15 @@ export class TestEndpoints {
   constructor({implementation, tag}) {
     this.implementation = implementation;
     this.tag = tag;
-    this.verifier = implementation.verifiers.find(
+    try {this.verifier = implementation.verifiers.find(
```
Comment (BigBlueHat) [inline tests/TestEndpoints.js:19]: @PatStLouis what about doing something like `this.verifier = this.verifier?.find(...) || null;`?

File: tests/TestEndpoints.js [hunk 2]
```diff
@@ -40,6 +43,9 @@ export class TestEndpoints {
     return result;
   }
   async verifyVp(vp, options = {checks: []}) {
+    if (this.vpVerifier === null) {
+      return null;
```
Comment (aljones15) [inline tests/TestEndpoints.js:47]: just throw here (unless in a before statement). That should result in the error report or at least the test runner telling them what is missing from their manifest. Please note 2 cases here: 1. they didn't add a vpVerifier section to their manifest 2. they added vpVerifiers but it had no matching tags and then case 3 they add a vpVerifier endpoint with some other missing prop such as tags. maybe we need a validator to run implementations on each endpoint.

File: tests/TestEndpoints.js (hunk 1 above)
Comment (PatStLouis) [reply tests/TestEndpoints.js]: @BigBlueHat I'm assuming you meant something like:
```
this.verifier = implementation.verifiers?.find(
      verifier => verifier.tags.has(tag)) || null;
```

Just tried it and it worked, I prefer this as a prettier solution, thank you

File: tests/TestEndpoints.js (hunk 2 above)
Comment (PatStLouis) [reply tests/TestEndpoints.js:44]: I will test these 3 use cases and report the current behavior

Comment (PatStLouis) [reply tests/TestEndpoints.js:44]: The problem is that negative tests are expecting an error (`assert.rejects`) so returning an error actually counts the test as passed, hence why I'm returning a null value

**Review by BigBlueHat — APPROVED** [review]

**Review by BigBlueHat — APPROVED** [review]

File: tests/TestEndpoints.js (hunk 2 above)
Comment (aljones15) [reply tests/TestEndpoints.js:44]: ahhh ok yeah I forgot that these test cases use a different assertion library so null is fine for now at least.

**Review by aljones15 — APPROVED** [review]
approving as this improves the project  alot and thanks for the improvements.

**Pull request merged**